apps/web/src/lib/components/ChatComponent.svelte

```diff
@@ -70,4 +87,40 @@
 			display: none;
 		}
 	}
+
+	.blank-state {
+		height: 100%;
+		display: flex;
+		align-items: center;
+		justify-content: flex-start;
+		padding-left: 30px;
+		margin-top: 20px;
+	}
+
+	.blank-state-content {
+		display: flex;
+		flex-direction: column;
+		align-items: left;
+		gap: 1rem;
+		text-align: left;
+		padding-left: 40px;
+	}
+
+	.blank-message {
+		padding-left: 17px;
+	}
+
+	.blank-message-title {
+		font-size: 1.3rem;
+		font-weight: 600;
+		color: var(--text-2, #333);
```

Comment:
`--text-2` isn't a CSS variable that we provide, you probably mean `--clr-text-2`